### DISCUSSION TITLE
The Project needs sponsors

### DISCUSSION THREAD

**Discussion by akulamartin** [UPBGE/upbge#1811] [Ideas/Features for UPBGE 0.3+]
Good day guys. I like the project and can't wait for mobile integration, though sponsors would greatly help.

**Comment by xcarlossilva** [comment]
Mobile integration or Web using HTML 5 is other interesting feature to add in the engine. I´m an artist and explore those platforms will be very good for knowledge and test different types of target public.
I guess this will increase the power of learning in the schools and in general, also. 
Publish in more platforms, please.
